docker/README.md

```diff
@@ -27,10 +28,14 @@ docker build --file Dockerfile.local --tag mira_dkg:latest .
 Once the build finished, you can run the container locally as
 
 ```shell
-docker run -d -p 8771:8771 -e MIRA_NEO4J_URL=bolt://0.0.0.0:7687 --name mira_dkg mira_dkg:latest
+# Option 1: run in the background
+docker run --detach -p 8771:8771 -e MIRA_NEO4J_URL=bolt://0.0.0.0:7687 --name mira_dkg mira_dkg:latest
```

Comment:
Given the domain-specific build, the docker build and docker run instructions should also be updated.